src/character_score.cpp

```diff
@@ -0,0 +1,240 @@
+#include "character.h"
```

Comment:
I don't know if there's any better name for this file; not sure about this one. @KorGgenT?

Reply:
There are a lot more `_modifier` than `_score` functions, so this makes sense to me.